Sources/TokamakCore/Views/Selectors/Slider.swift

```diff
@@ -0,0 +1,164 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public enum _SliderStep {
+  /* where Step: BinaryFloatingPoint.Stride */
+  public typealias Step = Double.Stride
+  case any
+  case discrete(Step)
+}
+
+/// A control for selecting a value from a bounded linear range of values.
+///
+/// Available when `Label` and `ValueLabel` conform to `View`.
+public struct Slider<Label, ValueLabel>: View where Label: View, ValueLabel: View {
+  let label: Label
+  let minValueLabel: ValueLabel
+  let maxValueLabel: ValueLabel
+  let valueBinding: Binding<Double>
+  let bounds: ClosedRange<Double>
+  let step: _SliderStep // <Double>
+  // “For example, on iOS, a Slider is considered to be actively editing while the user is touching the knob and sliding it around the track.”
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/Selectors/Slider.swift;
  Line: 32;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 142 characters (`line_length`)